src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -89,11 +88,11 @@
                                   (delete)="deleteInitiator()">
             </zac-bedrijfsgegevens>
 
-
             <mat-card class="zaken-card w50 flex-1">
                 <mat-card-header class="flex-row space-between">
                     <mat-card-title>{{zaak.identificatie}}</mat-card-title>
                     <mat-card-subtitle>{{zaak.zaaktype.omschrijving}}</mat-card-subtitle>
+                    <mat-icon *ngIf="zaak.isProcesGestuurd && zaak.isOpen && !zaak.isHeropend">process_chart</mat-icon>
                     <zac-zaak-indicaties [layout]="indicatiesLayout.VIEW" [zaak]="zaak"></zac-zaak-indicaties>
```

Comment:
Moet dit misschien een indicatie worden?

Reply:
Ik zou er geen indicatie van maken maar een knop waarop je kunt drukken om het proces diagram in te zien.